purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -1044,6 +1044,61 @@ internal class PurchasesOrchestrator(
         )
     }
 
+    @Suppress("CyclomaticComplexMethod")
+    fun setAppsFlyerAttributionData(data: Map<*, *>?) {
+        log(LogIntent.DEBUG) { AttributionStrings.METHOD_CALLED.format("setAppsFlyerAttributionData") }
+
+        if (data == null) {
+            return
+        }
+
+        val attributes = mutableMapOf<String, String?>()
+
+        val mediaSource = data.getStringValue("media_source")
+            ?: data.getStringValue("af_status")
+                .takeIf { it.equals("Organic", ignoreCase = true) }
+                ?.let { "Organic" }
```

Comment:
Hmm do we need to get `af_status` at all considering we only get it it if's `Organic` if we are falling back to `Organic` in any case?